tutorials/editor/upgrading_to_godot_4.rst

```diff
@@ -0,0 +1,544 @@
+| Light2D                                 | PointLight2D                              |
++-----------------------------------------+-------------------------------------------+
+| LineShape2D                             | WorldBoundaryShape2D                      |
++-----------------------------------------+-------------------------------------------+
+| Listener                                | AudioListener3D                           |
++-----------------------------------------+-------------------------------------------+
+| NavigationMeshInstance                  | NavigationRegion3D                        |
++-----------------------------------------+-------------------------------------------+
+| NavigationPolygonInstance               | NavigationRegion2D                        |
++-----------------------------------------+-------------------------------------------+
+| Navigation2DServer                      | NavigationServer2D                        |
++-----------------------------------------+-------------------------------------------+
+| PanoramaSky                             | Sky                                       |
++-----------------------------------------+-------------------------------------------+
+| Particles                               | GPUParticles3D                            |
++-----------------------------------------+-------------------------------------------+
+| Particles2D                             | GPUParticles2D                            |
++-----------------------------------------+-------------------------------------------+
+| ParticlesMaterial                       | ParticleProcessMaterial                   |
++-----------------------------------------+-------------------------------------------+
+| Physics2DDirectBodyState                | PhysicsDirectBodyState2D                  |
++-----------------------------------------+-------------------------------------------+
+| Physics2DDirectSpaceState               | PhysicsDirectSpaceState2D                 |
++-----------------------------------------+-------------------------------------------+
+| Physics2DServer                         | PhysicsServer2D                           |
++-----------------------------------------+-------------------------------------------+
+| Physics2DShapeQueryParameters           | PhysicsShapeQueryParameters2D             |
++-----------------------------------------+-------------------------------------------+
+| Physics2DTestMotionResult               | PhysicsTestMotionResult2D                 |
++-----------------------------------------+-------------------------------------------+
+| PhysicsDirectBodyState                  | PhysicsDirectBodyState3D                  |
++-----------------------------------------+-------------------------------------------+
+| PhysicsDirectSpaceState                 | PhysicsDirectSpaceState3D                 |
```

Comment:
This and DirectBodyState were only changed with a suffix being added and should be removed